backend/src/main/java/com/bakdata/conquery/models/query/concept/specific/CQOr.java

```diff
@@ -79,10 +83,20 @@ public void collectResultInfos(ResultInfoCollector collector) {
 		}
 
 		if (createExists) {
-			collector.add(new SimpleResultInfo(Objects.requireNonNullElse(getLabel(), "OR"), ResultType.BooleanT.INSTANCE));
+			collector.add(new SimpleResultInfo(getLabel(), ResultType.BooleanT.INSTANCE));
 		}
 	}
 
+	@Override
+	public String getLabel() {
+		return Objects.requireNonNullElse(super.getLabel(), QueryUtils.createDefaultMultiLabel(children, " " + getGetC10nName() + " "));
+	}
+
+	private static String getGetC10nName() {
+		return C10N.get(CQElementC10n.class, Locale.ROOT).or();
```

Comment:
Hier auch nicht Root

Reply:
Ohne Root habe ich hier deutsche Ausgabe lokal bekommen

Reply:
An der Stelle müsstest du irgendwie an die PrintSettings rankommen. Dort ist eine Locale drinne (In den Tests auf Englisch) oder I18n.LOCALE.get()

Reply:
Habe es als separaten Parameter hochgebubblet, kann mir aber auch vorstellen, dass wir das in PrintSettings stecken?

Reply:
Habe die PrintSettings als separaten Paremeter hochgebubbelt. Dadurch sind die an ein paar Stellen reingekommen wo sie evtl komisch sind